rero_ils/modules/documents/dojson/contrib/marc21tojson_kul/model.py

```diff
@@ -0,0 +1,1963 @@
+                'type': 'bf:Language'
+            })
+            lang_codes.append(lang_value)
+
+    # default provisionActivity if we have no 264
+    valid_260 = False
+    fields_260 = marc21.get_fields(tag='260')
+    if fields_260:
+        valid_260 = True
+
+    fields_264 = marc21.get_fields(tag='264')
+    valid_264 = False
+    for field_264 in fields_264:
+        valid_264 = valid_264 or field_264['ind2'] in ['0', '1', '2', '3']
+    if not valid_264 and not valid_260:
+        if fields_264:
+            error_print('WARNING INVALID 264', marc21.bib_id, marc21.rero_id,
+                        fields_264)
+        if fields_260:
+            error_print('WARNING INVALID 260', marc21.bib_id, marc21.rero_id,
+                        fields_260)
+        places = []
+        publication = {
+            'type': 'bf:Publication'
+        }
+        place = build_place()
+        if place:
+            places.append(place)
+        # parce le link skipping the fist (already used by build_place)
+        for i in range(1, len(marc21.links_from_752)):
+            place = {
+                'country': 'xx',
+                'type': 'bf:Place',
+                'identifyBy': marc21.links_from_752[i]
+            }
+            places.append(place)
+
+        if places:
+            publication['place'] = places
+        self['provisionActivity'] = [publication]
+
+        if (marc21.date_type_from_008 == 'q' or
+                marc21.date_type_from_008 == 'n'):
+            self['provisionActivity'][0][
+                'note'
+            ] = 'Date(s) uncertain or unknown'
+        start_date = make_year(marc21.date1_from_008)
+        if not start_date or start_date > 2050:
+            error_print('WARNING START DATE 008:', marc21.bib_id,
+                        marc21.rero_id, marc21.date1_from_008)
+            start_date = 2050
+            self['provisionActivity'][0][
+                'note'
+            ] = 'Date not available and automatically set to 2050'
+        self['provisionActivity'][0]['startDate'] = start_date
+        end_date = make_year(marc21.date2_from_008)
+        if end_date:
+            if end_date > 2050:
+                error_print('WARNING END DATE 008:', marc21.bib_id,
+                            marc21.rero_id, marc21.date1_from_008)
+            else:
+                self['provisionActivity'][0]['endDate'] = end_date
+        original_date = make_year(marc21.original_date_from_008)
+        if original_date:
+            if original_date > 2050:
+                error_print('WARNING ORIGINAL DATE 008:', marc21.bib_id,
+                            marc21.rero_id, marc21.original_date_from_008)
+            else:
+                self['provisionActivity'][0]['original_date'] = original_date
+    if not language:
+        error_print('ERROR LANGUAGE:', marc21.bib_id, 'set to "und"')
+        language = [{'value': 'und', 'type': 'bf:Language'}]
+    return language or None
```

Comment:
Probably you can return only `language` here: the `@ignore_value` should do the job.